tracker/view/edit.py

```diff
@@ -145,14 +145,14 @@ def edit_group(avg):
                   .filter(CVEGroup.id == group_id)
                   .join(CVEGroupEntry).join(CVE).join(CVEGroupPackage)
                   .outerjoin(Advisory, Advisory.group_package_id == CVEGroupPackage.id)
-                  .group_by(CVEGroup.id).group_by(CVE.id)
+                  .group_by(CVEGroup.id).group_by(CVE.id).group_by(CVEGroupPackage.pkgname)
                   .order_by(CVE.id)).all()
     if not group_data:
         return not_found()
 
     group = group_data[0][0]
-    issues = [cve for (group, cve, pkg, advisory) in group_data]
-    issue_ids = [cve.id for cve in issues]
+    issues = set([cve for (group, cve, pkg, advisory) in group_data])
```

Comment:
This is probably a cross-python compatibility issue, but I personally prefer

```python
issues = {cve for (group, cve, pkg, advisory) in group_data}
```

Reply:
we only support python3 anyway.... didn't even know this semantically means a set instead of a list but testing in a repl shows that it indeed does. Will consider for future change and standard, looks neat.